doc/source/whatsnew/v2.1.0.rst

```diff
@@ -244,7 +244,7 @@ Other
 
 .. ***DO NOT USE THIS SECTION***
 
--
+- Bug in :func:`assert_almost_equal` now throwing assertion error for two unequal sets (:issue:`51727`)
```

Comment:
This must be straight under the `Other` section, above the ` ***DO NOT USE THIS SECTION***` section 😄 .